package/Shaders/Water.hlsl

```diff
@@ -623,6 +635,12 @@ PS_OUTPUT main(PS_INPUT input)
 	                    finalSpecularColor;
 #			else
 	float3 sunColor = GetSunColor(normal, viewDirection);
+
+	// #				if defined(CLOUD_SHADOWS)
```

Comment:
Meant to leave this comment?

Reply:
Removed in https://github.com/doodlum/skyrim-community-shaders/pull/172/commits/72d637b4ba1091a722b991e0ba7dfb04dd58a321